app/common_handlers.py

```diff
@@ -544,35 +544,50 @@ async def post(self, request):
                 ))
 
         if address_confirmation == 'yes':
-
+            uprn_return = {}
+            have_uprn_return = False
             try:
-                census_address_type_value = attributes['censusAddressType']
-                if census_address_type_value == 'NA':
-                    logger.info('censusAddressType is NA',
-                                client_ip=request['client_ip'],
-                                client_id=request['client_id'],
-                                trace=request['trace'],
-                                user_selection=address_confirmation)
-                    raise HTTPFound(
-                        request.app.router['CommonCallContactCentre:get'].url_for(
-                            display_region=display_region, user_journey=user_journey, error='unable-to-match-address'))
-                elif (census_address_type_value == 'CE') and \
-                        (sub_user_journey == 'continuation-questionnaire'):
-                    logger.info('continuation form for a CE - rejecting',
-                                client_ip=request['client_ip'],
-                                client_id=request['client_id'],
-                                trace=request['trace'],
-                                sub_journey=sub_user_journey,
-                                census_addr_type=census_address_type_value)
-                    raise HTTPFound(
-                        request.app.router['RequestContinuationNotAHousehold:get'].url_for(
-                            display_region=display_region))
-            except KeyError:
-                logger.info('unable to check censusAddressType',
-                            client_ip=request['client_ip'], client_id=request['client_id'], trace=request['trace'])
+                uprn_return = await RHService.get_case_by_uprn(request, attributes['uprn'])
+                attributes['censusAddressType'] = uprn_return['addressType']
```

Comment:
I may have missed it , but I can't see where you handle the condition where RM finds the case , but returns AddressType=NA. Then you need to convert it to addressType=HH.   This is the 3rd item in the table in JIRA.